challenge_manipulation/src/manipulation.py

```diff
@@ -45,7 +45,7 @@
 from empty_shelf_designator import EmptyShelfDesignator
 
 # PDF writer
-import pdf
+# import pdf
```

Comment:
It even says so in the commit message: 
https://github.com/tue-robotics/tue_robocup/commit/ab3cb997b41bb4613a417532e54d29ea72330417